NEWS.txt

```diff
@@ -51,6 +51,20 @@ restore snapshots created with the previous major version using the
 'sstableloader' tool. You can upgrade the file format of your snapshots
 using the provided 'sstableupgrade' tool.
 
+4.0.8
+=====
+
+New features
+------------
+
+Upgrading
+---------
+    - All previous versions of 4.x contained a mistake on the implementation of the old CQL native protocol v3. That
+     mistake produced issues when paging over tables with compact storage and a single clustering column during rolling
+     upgrades involving 3.x and 4.x nodes. The fix for that issue makes that it can now appear during rolling upgrades
+     from 4.0.0-4.0.7. If that your case, please use protocol v4 or higher in your driver. See CASSANDRA-17507 for
```

Comment:
- `The fix for that issue makes that it can now appear` -> `The fix for that issue makes ***it that*** it can now appear`?
- `if that _is/were_ your case`?